R/bind_ard.R

```diff
@@ -7,10 +7,18 @@
 #'   ARDs to combine. Each argument can either be an ARD,
 #'   or a list of ARDs. Columns are matched by name, and any missing
 #'   columns will be filled with `NA`.
+#' @param .distinct (`logical`)\cr
+#'   logical indicating whether to remove non-distinct values from the ARD.
+#'   Duplicates are checked across grouping variables, primary variables,
+#'   context (if present), the **statistic name and the statistic value**.
+#'   Default is `FALSE`. If a statistic name is repeated and `.update=TRUE`,
+#'   the more recently added statistics will be retained, and the other(s) omitted.
 #' @param .update (`logical`)\cr
-#'   logical indicating whether to update duplicate ARD statistics.
-#'   Default is `FALSE`. If a statistic type is repeated and `.update=TRUE`,
-#'   the more recently added statistics will be retained, and the others omitted.
+#'   logical indicating whether to update ARD and remove duplicated named statistics.
+#'   Duplicates are checked across grouping variables, primary variables, and the
+#'   **statistic name**.
+#'   Default is `FALSE`. If a statistic name is repeated and `.update=TRUE`,
```

Comment:
```suggestion
#'   Default is `FALSE`. If a statistic name is repeated and `.update = TRUE`,
```

Reply:
For the help file, i like keeping the code a bit shorter, so i'll just keep it as it is